.storybook/main.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  core: {
```

Comment:
Needed to upgrade to webpack 5 in order to resolve issues with Node 18 and SSL. See: https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported